### PR TITLE
Add homepage

### PR DESCRIPTION
**Pull request by gnuion**
In this pull request I:
- Added readme file.
- Added HTML and CSS files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+> This is a demo project for which I set up GitHub Actions without knowing a lot about it.
+
+## Built With
+
+- HTML, CSS
+
+## Getting Started
+
+To clone this project use `git clone`.
+
```
Comment (danmainah) [inline README.md:14]: - [ ] [optional] Kindly add the requirement and how to clone and run your project locally.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (danmainah) [inline .gitignore:1]: - [ ] Kindly make sure to add the node_modules directory in your .gitignore file, it's a project requirement.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @gnuion , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used

 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] Kindly use this pull request when requesting for review, do not create another project or pul request , this will make the reviewers task easier.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 2 above)
Comment (gnuion) [reply .gitignore:1]: There is no node_module directory because I don't use nodejs at all!

**Review by danmainah — APPROVED** [review]
Hi @gnuion , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gnuion** [comment]
🎉

**Pull request closed without merging**